FileBaseContext/Serializers/CsvRowDataSerializer.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace kDg.FileBaseContext.Serializers;
+
+public class CsvRowDataSerializer : IRowDataSerializer
```

Comment:
This type could optionally be `internal`

Reply:
fixed in 1a5c0c51b15ac1c2e0fd7f85123225f2f5956835